radixdlt-core/radixdlt/src/main/java/com/radixdlt/client/api/ValidatorInfoDetails.java

```diff
@@ -0,0 +1,88 @@
+/*
+ * (C) Copyright 2021 Radix DLT Ltd
+ *
+ * Radix DLT Ltd licenses this file to you under the Apache License,
+ * Version 2.0 (the "License"); you may not use this file except in
+ * compliance with the License.  You may obtain a copy of the
+ * License at
+ *
+ *  http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND,
+ * either express or implied.  See the License for the specific
+ * language governing permissions and limitations under the License.
+ */
+
+package com.radixdlt.client.api;
+
+import org.json.JSONObject;
+
+import com.radixdlt.identifiers.RadixAddress;
+import com.radixdlt.utils.UInt256;
+
+import static org.radix.api.jsonrpc.JsonRpcUtil.jsonObject;
+
+import static java.util.Objects.requireNonNull;
+
+public class ValidatorInfoDetails {
+	private final RadixAddress address;
```

Comment:
better change this to `String address` for now? because it will be `ValidatorAddress` class later, so letting it be `RadixAddress` is just misleading and breaks API? Will will be sending arbitrary string from JS

Reply:
or `ECPublicKey`